### PR TITLE
Add python-like function print

### PR DESCRIPTION
**Pull request by dShvetsov**
Print can prints. And no more. Only print. You put variables into print, and it prints it. Split output by spaces, add newline at the end. And nothing more.

### CODE REVIEW

**Comment by AlexandrShcherbakov** [comment]
This function can not be used without arguments. It needs tests. Brackets style should be similar to style in other files. Usage of printf should be faster, I think.
Please, fix these issues.

File: include/Print.h [hunk 1]
```diff
@@ -8,17 +8,72 @@ namespace PySTL
 // TODO [dshvetsov] : Add support of output specification
 //                    (such set_precision, set_base, etc).
 
+namespace detail {
+
+template <typename Head, typename... Args>
+class __print_impl
```
Comment (AlexandrShcherbakov) [inline include/Print.h]: Don't use "__" in your variable and class names. Names with "__" reserved for stl.

File: include/Print.h [hunk 2]
```diff
@@ -8,17 +8,72 @@ namespace PySTL
...
+
+private:
+    __print_impl<Args...> m_data;
+    std::ostream& m_out = std::cout;
+//    std::string delim = " ";
+//    std::string endline = "\n";
+};
+
+
+} // namespace deatail
```
Comment (AlexandrShcherbakov) [inline include/Print.h]: Typo s/deatail/detail/

File: include/Print.h [hunk 3]
```diff
@@ -93,7 +93,7 @@ class __print_holder {
     std::ostream* m_out = &std::cout; // Cannot use refernce
                                      // becouse it may be replaced
     std::string_view m_sep = " ";
-    std::string_view m_end = " ";
+    std::string_view m_end = "\n";
```
Comment (AlexandrShcherbakov) [inline include/Print.h]: This end of line is platform specific.

File: include/Print.h [hunk 4]
```diff
@@ -8,17 +8,72 @@ namespace PySTL
...
+
+template <typename Head, typename... Args>
+class __print_impl
+{
+public:
+    __print_impl(const Head& head, const Args&... tail)
+        : m_head(head), m_tail(tail...)
+    { }
+
+    void flush(std::ostream& out)
```
[the remark this answers is not in the document]
Comment (dShvetsov) [reply include/Print.h]: I tried add opportunity use named arguments.
I can add operator(), but want to this approach stay too.
Could you give me your advises in order of this arguments.

File: include/Print.h (hunk 3 above)
Comment (dShvetsov) [reply include/Print.h]: I haven't found proper way to use platform specific endline character(s).

std::endl prints '\n' too.

**Comment by dShvetsov** [comment]
I enabled print without arguments, and added some test. What brackets style do your prefer, and how I should organize print in directories?

**Pull request merged**